scripts/env_create.sh

```diff
@@ -90,6 +90,7 @@ fi
 # Activate and install packages in the environment
 echo "Installing basic build requirements."
 if [[ $include_build_deps == 1 ]]; then
+    conda install -y numpy cmake
```

Comment:
We assume `cmake` is already installed on the system. `numpy` is already being installed by the command on the next line.

Reply:
numpy isn't bulding with pip on m1 right now (as multiple other repos and my own experiments show 📦 ), cmake is bundled with conda so dunno why not just pull it with script we can check if its installed and just ignore it then 😄